test/02-investigate.t

```diff
@@ -132,13 +134,35 @@ clone_call=echo
 _clone_call() {
     echo "$@" >&2
 }
+
+fetch-vars-json() {
+    local tid=$1
+    if [ $1 == 10020 ]; then
+	echo '{"TEST_GIT_URL": "https://github.com/os-autoinst/os-autoinst-distri-openQA.git"}'
```

Comment:
No tabs please, also two lines further down.
I'm not sure if shellcheck can be configured to complain about tabs.

Reply:
I reported https://progress.opensuse.org/issues/153793